templates/newsletter.html

```diff
@@ -1,11 +1,16 @@
 <section data-component="n-newsletter-signup" data-newsletter-id="{{id}}" data-newsletter-name="{{name}}" class="n-newsletter-signup" {{#if isPremium}}data-newsletter-is-premium{{/if}}>
 	<div class="n-newsletter-signup__header">
-		<span class="n-newsletter-signup__heading">{{name}}<span class="n-util-visually-hidden">{{#if isPremium}}, premium{{/if}}{{#if frequency}}, {{frequency}}{{/if}}</span></span>
-		{{#if isPremium}}<span aria-hidden="true" class="n-newsletter-signup__top-meta o-labels o-labels--premium">Premium</span>{{/if}}
+		<div class="n-newsletter-signup__heading">
+			<div class="n-newsletter-signup__heading__mail-icon__wrapper">
+				<span class="n-newsletter-signup__heading__mail-icon"/>
+			</div>
+			<span class="n-newsletter-signup__heading__title">{{name}} Email<span class="n-util-visually-hidden">{{#if isPremium}}, premium{{/if}}{{#if frequency}}, {{frequency}}{{/if}}</span></span>
```

Comment:
Given it will initially be used on an article page, and there is a `h2` with the text "Listen to this article" before it in the page, I would have this as a `h2`.

If you plan to have this used in other pages, you could make `h2` the default heading level, and provide a parameter to the template to override it if needed.